lib/build.gradle.kts

```diff
@@ -57,10 +57,15 @@ val buildCryptoCpp = task<Exec>("BuildCryptoCpp") {
     commandLine("${project.projectDir}/build_crypto_cpp.sh")
 }
 
-// For tests we simply use version from crypto build
+val compileContracts = task<Exec>("compileContracts") {
+    commandLine("${project.projectDir}/src/test/resources/compileContracts.sh")
```

Comment:
Update readme to inform that starknet cli is needed now.

Reply:
It's only required to run tests, but I'll add that to readme.